modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -4,10 +4,20 @@ Applications and API gateways deployed to Runtime Fabric include native support
 
 == Anypoint Monitoring
 
-xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components 
-in your application network. For applications and API gateways deployed to Runtime Fabric, Anypoint Monitoring 
-provides basic metrics to all customers. Titanium customers have access to additional capabilities including 
-application log retrieval and search.
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule runtime engine (Mule) flows and components in your application network. For applications and API gateways deployed to Anypoint Runtime Fabric (Runtime Fabric), Anypoint Monitoring provides basic metrics to all customers. Titanium-plan customers have access to additional capabilities, including application log retrieval and search.
+
+=== Enable Anypoint Monitoring
```

Comment:
...and now that I have read the topic, this really isn't about enabling, because that is the default.
It is about ensuring the prerequisite of an outbound connection is available. 
Consider tweaking the wording to clarify, because users really don't need to enable monitoring.